src/extension.ts

```diff
@@ -105,26 +134,20 @@ async function findLookupPath(): Promise<string | undefined> {
       }
       return undefined;
     });
-  }
 
-  if (gitRoot) {
     return gitRoot;
```

Comment:
@solaris765 This is where gitRoot returns

Reply:
ah, my mistake. I see it now. We're having an issue now with scopes from the .cz-config not being found and this looked related. I'll troubleshoot on my own. Thanks pointing this out for me.

Reply:
@solaris765 I am using an extension of the current version in a monorepository and with multiple repositories. There were no problems with the config. Sometimes it happened that I did not find it. Restarting VS Code helped

Reply:
Got it. I'll take a look at whats going on then, we have a similar setup but store our config at the git root. Could very well just need a restart though.